src/tests/class_tests/openms/source/TargetedSpectraExtractor_test.cpp

```diff
@@ -44,6 +44,20 @@
 using namespace OpenMS;
 using namespace std;
 
+vector<MSSpectrum>::const_iterator find_spectrum_by_name(const vector<MSSpectrum>& spectra, const String& name)
```

Comment:
chagne to findSpectrumByName

Reply:
Is it the preferred way of naming non-class functions? @timosachsenberg

Reply:
I think so. At least we also use that for static functions...